src/test/kotlin/com/github/avrokotlin/avro4k/schema/EnumSchemaTest.kt

```diff
@@ -48,4 +97,16 @@ enum class Wine {
 @AvroDoc("documentation")
 enum class Suit{
    SPADES, HEARTS, DIAMONDS, CLUBS;
-}
\ No newline at end of file
+}
+
+@Serializable
+@AvroEnumDefault("MEAT")
+enum class IngredientType { VEGGIE, MEAT, }
+
+@Serializable
+@AvroEnumDefault("MEAT")
```

Comment:
Annotating a data class with an AvroEnumDefault should have no impact on the gernerated schema for the data class, as the AvroEnumDefault annotation should only be evaluated for enums. Is this what you want to test with this case? If so, please clarify it in the test name.

Reply:
Actually I just removed this test. Or it was something that @darmstrong1 added to test field defaults. But with the new enum defaults annotation and the original avrodefault annotation probably it sufficient enough that you don't need to anymore add that additional layer / field?